sources/addr.json

```diff
@@ -3,8 +3,10 @@
 	"fullname": "addr",
 	"name": "addr",
 	"prefix": "addr",
-    	"key":"id",
+	"key": "id",
 	"url": "",
-	"license": "CC0",
-	"description": ""
-}
+	"license": "N/A",
+	"license_type": "CC0",
+	"license_text": "",
+	"description": "Property prefix."
```

Comment:
I'm comfortable with "property prefix", since this is the `whosonfirst-sources` repo.. but I can file an issue to update these to read "WOF property prefix" if we decide that's a better approach once/if we add more prefix sources.